internal/commands/result_test.go

```diff
@@ -372,6 +373,31 @@ func TestRunGetResultsByScanIdGLFormat(t *testing.T) {
 	os.Remove(fmt.Sprintf("%s.%s", fileName, printer.FormatGL))
 }
 
+func TestRunGetResultsByScanIdGLFormat_NoVulnerabilities_Success(t *testing.T) {
+	// Execute the command and perform nil assertion
+	execCmdNilAssertion(t, "results", "show", "--scan-id", "MOCK_NO_VULNERABILITIES", "--report-format", "gl-sast")
+
+	// Run test for gl-sast report type
+	// Check if the file exists and vulnerabilities is empty, then delete the file
+	if _, err := os.Stat(fmt.Sprintf("%s.%s-report.json", fileName, printer.FormatGL)); err == nil {
+		t.Logf("File exists: %s.%s", fileName, printer.FormatGL)
+		resultsData, err := os.ReadFile(fmt.Sprintf("%s.%s-report.json", fileName, printer.FormatGL))
+		if err != nil {
+			t.Fatalf("Failed to read file: %v", err)
```

Comment:
do we want fatal in tests ? it the assertion is pass let's have a warning log maybe